src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/ContentRepository.kt

```diff
@@ -32,6 +35,17 @@ class ContentRepository(
                 .subscribeOn(Schedulers.io())
     }
 
+    override fun getCollectionMetaContent(collection: Collection): Single<Content> {
+        return Single
+            .fromCallable {
+                contentDao
+                    .fetchByCollectionIdAndStart(collection.id, 1, listOf(ContentDao.Labels.CHAPTER))
```

Comment:
Why is the start parameter 1 here? Perhaps I don't remember what this meta content is.

Reply:
The parameter is for the start, but this is the metacontent so the start would always be 1

Reply:
This line will also be replaced to select `ContentType.META` (in another PR).